README.md

```diff
@@ -1150,23 +1152,3 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
     do_something
   end
   ~~~
-
-## The Rest
-
-* Avoid long methods.
-
-* Avoid long parameter lists.
-
-* Avoid needless metaprogramming.
-
-* Avoid using `update_all`. If you do use it, use a scoped association
```

Comment:
Removing this rule since it is Rails specific.

Reply:
Feels like there may be value in us creating a Rails style guide. Similarly to how we release this as the `rubocop-shopify` gem and depend on `rubocop`, we could have a `rubocop-rails-shopify` 🤔 

But yeah, I'm fine this style guide being strictly about Ruby.